src/cmd/commit.ts

```diff
@@ -69,7 +69,7 @@ export const cloneOtomiChartsInGitea = async (): Promise<void> => {
   const values = (await hfValues()) as Record<string, any>
   const { email, username, password } = getRepo(values)
   const workDir = '/tmp/otomi-charts'
-  const otomiChartsUrl = 'https://github.com/redkubes/otomi-charts.git'
+  const otomiChartsUrl = process.env.OTOMI_CHARTS_URL || 'https://github.com/redkubes/otomi-charts.git'
```

Comment:
Ferruh, take a look at`src/validators.ts` file.
The env type and its default should be defined there.